zsh-ansimotd.plugin.zsh

```diff
@@ -52,14 +52,20 @@ function ansi_art_random_file {
 function ansi_art_random {
   ansi_filename="$(ansi_art_random_file)"
 
+  # optional use pv to limit rate we output
+  viewer=(cat)
+  if (( $+commands[pv] )); then
```

Comment:
Can we change this to opt in, given some people may have `pv` installed but may not want `pv` buffered output, although cool as hell i'm not sure on the consensus if it's an expected default/always on (when pv is installed) at this point so i'd like err on the side of user choice

In the condition let's add another ENV which will need to be satisfied to enable this output perhaps something like `ANSI_MOTD_RATE_LIMIT_OUTPUT`, suggestions welcome on alternative naming!

Reply:
Yeah, totally agree. :+1: Updated to use the suggested name and defaulted the rate to 1tbps. ;)